frontend/packages/console-app/src/components/quick-starts/QuickStartConfiguration.tsx

```diff
@@ -122,7 +122,7 @@ const QuickStartConfiguration: React.FC<{ readonly: boolean }> = ({ readonly })
       spec: {
         customization: {
           quickStarts: {
-            disabled,
+            disabled: disabled || [],
```

Comment:
I think it is better to update the `disabled` type to match the `fireTelemetryEvent` type definition, which is ` disabled: string[] | undefined;`. 
Remove the default `[]`, then add `undefined`  to the type definition in line 28 - ` disabled: string[]  | undefined;`. That should fix the type error.